cspatients/views.py

```diff
@@ -81,14 +82,24 @@ def post(self, request):
 
 class CheckPatientExistsView(APIView):
     def post(self, request):
+        patient_data = {}
+        return_status = status.HTTP_200_OK
+
         try:
             patient_id = get_rp_dict(request.data)["patient_id"]
-            PatientEntry.objects.get(
+            patient_entry = PatientEntry.objects.get(
                 patient__patient_id=patient_id, completion_time__isnull=True
             )
+
+            patient_entry_serializer = PatientEntrySerializer(patient_entry)
+            patient_serializer = PatientSerializer(patient_entry.patient)
+
+            patient_data = patient_entry_serializer.data
+            patient_data.update(patient_serializer.data)
```

Comment:
@erikh360 Please why did you have to update the patient_data variable after passing into it the data from the serializer. I was thinking you might do something like patient_data = patient_entry_serializer.data + patient_serializer.data and then return patient_data as a json.  
Or is this a better way of doing the same thing.

Reply:
I tried the `+` like the example but you can't merge two dicts like that.

I set `patient_data` to the `PatientEntry` serialised data, then update it with the `Patient` serialised data. Two different models and two different serializers.